### PR TITLE
Add JOSS and Zenodo badges to doc

### PR DESCRIPTION
**Pull request by ahartikainen**
Simple fix.

### CODE REVIEW

**Comment by aloctavodia** [comment]
We may also want to add a short entry like in  #544

If you use ArviZ and want to cite it please use [DOI](https://doi.org/10.21105/joss.01143)

Here is the citation in BibTeX format

```
@article{arviz_2019,
	title = {{ArviZ} a unified library for exploratory analysis of {Bayesian} models in {Python}},
	author = {Kumar, Ravin and Colin, Carroll and Hartikainen, Ari and Martin, Osvaldo A.},
	journal = {The Journal of Open Source Software},
	year = {2019},
	doi = {10.21105/joss.01143},
	url = {http://joss.theoj.org/papers/10.21105/joss.01143},
}
```

**Comment by ahartikainen** [comment]
I added notation about citation. Do you know if there is a way to force newline? Currently I use different blocks between `If you use ArviZ and want to cite it ....` and `Here is ....`.

**Comment by canyon289** [comment]
I think `| `forces newline in rst

**Comment by canyon289** [comment]
I also have fix for testing failure in my docker PR

**Comment by canyon289** [comment]
Ok, wont have time today, might get to it tomorrow or next day

**Comment by ahartikainen** [comment]
No worries, this can wait :)

**Comment by canyon289** [comment]
Rebased on top of master

**Comment by canyon289** [comment]
One or two more days and will get to this

File: doc/index.rst [hunk 1]
```diff
@@ -38,6 +46,21 @@ Contributions and issue reports are very welcome at
...
 
+If you use ArviZ and want to cite it please use |JOSS|
+
+Here is the citation in BibTeX format  
+
+.. code:: bash
+
+    @article{arviz_2019,
+	    title = {{ArviZ} a unified library for exploratory analysis of {Bayesian} models in {Python}},  
+	    author = {Kumar, Ravin and Colin, Carrol and Hartikainen, Ari and Martin, Osvaldo A.},
```
Comment (aloctavodia) [inline doc/index.rst]: typo: Carroll

**Pull request merged**